File: nom-ui/src/app/core/models/meal-plan-entry.model.ts
export interface MealPlanEntry {
  id: number;
  recipeId: number | null;
  recipeName: string | null;
  recipeImage: string | null;
  title: string | null;
  notes: string | null;
  calories: number | null;
  proteinGrams: number | null;
  carbGrams: number | null;
  fatGrams: number | null;
  completedDate: string | null;
}

File: nom-api/Nom.Orch/Models/MealPlan/MealPlanEntryModel.cs
using System;

namespace Nom.Orch.Models.MealPlan
{
    public class MealPlanEntryModel
    {
        public long Id { get; set; }
        public long? RecipeId { get; set; }
        public string? RecipeName { get; set; }
        public string? RecipeImage { get; set; }
        public string? Title { get; set; }
        public string? Notes { get; set; }
        public decimal? Calories { get; set; }
        public decimal? ProteinGrams { get; set; }
        public decimal? CarbGrams { get; set; }
        public decimal? FatGrams { get; set; }
        public DateOnly? CompletedDate { get; set; }
    }
}
